tests/spec/modules/options/option-item-boolean.spec.js

```diff
@@ -0,0 +1,46 @@
+
+import { getOptionItemWith } from './index';
+
+describe("spec OptionItem:", () => {
+
+  describe("which has boolean value: ", () => {
+
+    beforeEach(() => {
+      this.optionItem = getOptionItemWith('boolean');
+      document.body.innerHTML = __html__['options.html'];
+    })
+
+    it("throw a error when string is fed into assertValue", done => {
+      try {
+        this.optionItem.assertValue('stststring');
+      } catch (e) {
+        done();
+      }
+    });
+
+    it("nothing when boolean is fed into assertValue", done => {
+      try {
+        this.optionItem.assertValue(false);
+        done();
+      } catch (e) {
+        fail(e);
+      }
+    });
+
+    it("ele.checked expected the value when setValueOnElement is called with a value", () =>{
```

Comment:
- change to `return a reversed value when setValueOnElement is called with the value.`